components/backup-stream/src/endpoint.rs

```diff
@@ -632,6 +551,7 @@ where
             .await
             .map_err(|err| Error::from(err).report("failed to get tso from pd"))
             .unwrap_or_default();
+        cm.update_max_ts(pd_tso);
```

Comment:
why move this line?

Reply:
we should update this to pd ts before we trying to resolve, so there won't be new async-commit prewrite uses [0, pd_ts] to commit during we resolving.